pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[tool.poetry]
+name = "scvi-tools-gui"
+version = "0.0.1"
+description = "A gui for scvi-tools."
+authors = ["Michael Jayasuriya <mjayasur@berkeley.edu>"]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Intended Audience :: Science/Research",
+  "Natural Language :: English",
+  "Programming Language :: Python :: 3.6",
+  "Programming Language :: Python :: 3.7",
+  "Operating System :: MacOS :: MacOS X",
+  "Operating System :: Microsoft :: Windows",
+  "Operating System :: POSIX :: Linux",
+  "Topic :: Scientific/Engineering :: Bio-Informatics",
+]
+license = "BSD-3-Clause"
+
+[tool.poetry.dependencies]
+dash = ">=1.19.0"
+dash-bootstrap-components = "0.11.2"
+plotly = ">=4.14.3"
+python = ">=3.6.2,<4.0"
+scanpy = ">=1.6"
+scikit-misc = ">=0.1.3"
+scvi-tools = ">=0.8.1"
+
+[tool.poetry.dev-dependencies]
+
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
```

Comment:
```suggestion
[build-system]
build-backend = "poetry.masonry.api"
requires = [
  "poetry>=1.0",
  "setuptools", # keep it here or "pip install -e" would fail
]
```